text/chapter5.md

```diff
@@ -110,10 +110,11 @@ In this case, the third line uses a guard to impose the extra condition that the
 
 As this example demonstrates, guards appear on the left of the equals symbol, separated from the list of patterns by a pipe character (`|`).
 
- ## Exercises
+## Exercises
 
- 1. (Easy) Write the factorial function using pattern matching. _Hint_. Consider the two cases zero and non-zero inputs.
- 1. (Medium) Look up _Pascal's Rule_ for computing binomial coefficients. Use it to write a function which computes binomial coefficients using pattern matching.
+1. (Easy) Write the `factorial` function using pattern matching. _Hint_: Consider the two corner cases of zero and non-zero inputs. _Note_: This is a repeat of an example from the previous chapter, but see if you can rewrite it here on your own.
+1. (Medium) Write a function `binomial` which finds the coefficient of the x^`k`th term in the polynomial expansion of (1 + x)^`n`. This is the same as the number of ways to choose a subset of `k` elements from a set of `n` elements. Use the formula `n! / k! (n - k)!`, where `!` is the factorial function written earlier. _Hint_: Use pattern matching to handle corner cases.
+1. (Medium) Write a function `pascal` which uses [_Pascal`s Rule_](https://en.wikipedia.org/wiki/Pascal%27s_rule) for computing the same binomial coefficients as the previous exercise.
```

Comment:
@dvail I noticed your "Pascal's Rule" solution uses the binomial formula with factorial. I split this into two exercises to emphasize the differences between the two. Not sure if there's much of a benefit to having two of these though. It may also be better to replace these exercises with less math-intensive ones that still demonstrate pattern matching.

Reply:
Sounds good, thanks for the review. I like the idea of shying away from math-intensive exercises, as someone without a math background might get stuck more on the subject matter than their understanding of the language constructs.

Is similarity to the original text a goal of the community maintained project? I've tried to keep the PRs I've worked as close to the original as possible, but if we are looking for general improvements as well I'll keep that in mind.

Reply:
> Is similarity to the original text a goal of the community maintained project?

I wouldn't say so. I believe we should prioritize clarity for readers and keep things up to date with the current language ecosystem, even if that involves significant rewrites.

We can always revert controversial changes.